strictdoc/export/html/renderers/markup_renderer.py

```diff
@@ -141,7 +143,7 @@ def render_node_field(
         output = self.fragment_writer.write(parts_output)
         self.cache[(document_type, node_field, truncated)] = output
 
-        return output
+        return Markup(output)
```

Comment:
Let's add a comment here that explains that docutils escapes stuff by itself.

Reply:
Let's add a type hint to the .write() methods that it turns Markup, not str to make it explicit in the type system that the stuff is escaped already?

Reply:
Sounds good. I have been moving to 100% happy mypy when it works in `--strict` mode.